resolwe_api/resolwe.py

```diff
@@ -250,7 +94,7 @@ def print_process_inputs(self, process_name):
         for field_schema, _, _ in iterate_schema({}, p['input_schema'], 'input'):
             name = field_schema['name']
             typ = field_schema['type']
-            sys.stdout.write("{} -> {}".format(name, typ))
+            sys.stdout.write("{} -> {}".format(name, typ) + "\n")
```

Comment:
Why `+ "\n"` instead of just modifying the original string by adding a `\n`, so `"{} -> {}\n".format(name, typ)`.